CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+1.4.0 / 2019-08-28
+==================
+
+###### Changes
+* Dropping support for PHP 5.3
+* Updated documentation about testing
+* Syn library is [0.0.2](https://github.com/bitovi/syn/tree/v0.0.2)
```

Comment:
Please move into `Misc` sub-section and replace with:

```
* Upgraded Syn library (used to emulate JavaScript events) to 0.0.3 version
```

---

I also was curious how it can be, that 7b7b29fb4f9b7fb0366b31011611ac47191373c3 comment mentions 0.0.3 version, but `syn.js` file has 0.0.2 version inside.

Then I've compared source code of `syn.js` file with official 0.0.3 release from https://raw.githubusercontent.com/bitovi/syn/v0.0.3/dist/syn.min.js and they've matched.

Reply:
updated.
also I've updated version in syn.js to avoid mistakes